fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1333,6 +1333,30 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long memory_tracker_interval_seconds = 60;
 
+    /**
+     * true to enable collect proc memory alloc profile
+     */
+    @ConfField(mutable = true)
+    public static boolean proc_profile_enable = true;
+
+    /**
+     * The number of seconds between proc profile collections
+     */
+    @ConfField(mutable = true)
+    public static long proc_profile_collect_interval_s = 3600;
```

Comment:
300 by default?

Reply:
+1. 3600 it too long to catch spikes

Reply:
done
default interval is 600 and collect time is 300 because the cpu and mem cannot be collected at the same time.